edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java

```diff
@@ -46,7 +46,7 @@ public CreateTestTAResource() {
             // since the sentence below are not properly tokenized
             Map<String, String> nonDefaultValues = new HashMap<>();
             nonDefaultValues.put(CuratorConfigurator.RESPECT_TOKENIZATION.key,
-                    CuratorConfigurator.TRUE);
+                    CuratorConfigurator.FALSE);
```

Comment:
So after fixing this, would it make sense to add this example which was failing somewhere in a unit test? 
https://github.com/mssammon/illinois-cogcomp-nlp-1/blob/master/edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java#L86-L87